experimental/dds/tree2/src/feature-libraries/sequence-field/rebase.ts

```diff
@@ -315,15 +305,14 @@ function rebaseMark<TNodeChange>(
 				return { count: 0 };
 			}
 
-			const nodeChange = getNodeChange(rebasedMark);
+			const nodeChange = rebasedMark.changes;
 			if (nodeChange !== undefined) {
 				rebasedMark = withNodeChange(rebasedMark, undefined);
-				const modify: Modify<TNodeChange> = {
-					type: "Modify",
+				const nestedChange: NoopMark<TNodeChange> = {
```

Comment:
It seems like the name `nestedChange` is misleading, since this is a mark containing a nested change.

Reply:
Fair enough, I thought "modify" was weird because it's not modifying the field, but you're right that this is even more confusing. I will keep "modify".